invenio/base/bundles.py

```diff
@@ -133,11 +133,23 @@
 
 admin = Bundle(
     "js/admin.js",
+    "vendors/admin-lte/dist/js/app.min.js",
     output="admin.js",
     filters=RequireJSFilter(exclude=[jquery]),
     weight=50
 )
 
+adminnextgen = Bundle(
+    "vendors/admin-lte/dist/css/AdminLTE.min.css",
+    "vendors/admin-lte/dist/css/skins/skin-blue.min.css",  # change for other skin
```

Comment:
We have to define at least some basic colors for the sidebar, navigationbar, and so on. But we could create a default skin `skin-default` to define them using some of the invenio colors for example.